api/openapi.gen.json

```diff
@@ -48,6 +48,7 @@
         },
         "type": "object"
       },
+      "v1.InstanceType": {},
```

Comment:
The generated type (component) is empty, this is because you need to add Go tags to the type in order to have the field exported:

https://github.com/adiabramovitch/provisioning-backend/blob/cf19486b4b50f3e19234dc8d0c0b53167a62d566/internal/payloads/instance_types_payload.go#L11